site/en/r2/tutorials/load_data/text.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>I think this is much cleaner than creating functions within functions</p><p><br></p><p>```</p><p>def labeler(example, index):</p><p>&nbsp;return example, tf.cast(index, tf.int64)&nbsp;&nbsp;</p><p><br></p><p>labeled_data_sets = []</p><p>for i, file_name in enumerate(FILE_NAMES):</p><p>&nbsp;lines_dataset = tf.data.TextLineDataset(file_name)</p><p>&nbsp;labeled_dataset = lines_dataset.map(lambda ex: labeler(ex, i))</p><p>&nbsp;labeled_data_sets.append(labeled_dataset)</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>